lib/ansible/modules/cloud/azure/azure_rm_appgateway.py

```diff
@@ -278,8 +284,8 @@
                     - 'disabled'
             request_timeout:
                 description:
-                    - "Request timeout in seconds. Application Gateway will fail the request if response is not received within RequestTimeout. Acceptable va
-                      lues are from 1 second to 86400 seconds."
+                    - Request timeout in seconds. Application Gateway will fail the request if response is not received within RequestTimeout. Acceptable va
+                      lues are from 1 second to 86400 seconds.
```

Comment:
If you accept the suggestion above, adding this content to another line, remove it here:
```suggestion
```